config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -55,17 +55,17 @@ susy.quest.ql.26.desc=There are a few creatures you may encounter.
 susy.quest.db.1.title=It's JEI, not NEI. Use it wisely.
 susy.quest.db.1.desc=It is probably impossible to try to play GregTech without using JEI. So here are some tips on how to use JEI to its full potential.%n%nThis pack will assume that the player's IQ is above 100. If you are confused, press §6R §ron an item in the questbook to see the item's crafting recipe, or §6U §rto see its uses.%n%n§eBookmark items regularly§r (default key §aA§r) so you don't have to type the names over and over.%n%nMost of the items in this pack have §emultiple recipes§r! Don't stop at the first recipe and think that item is impossible.%nSometimes a material is only available from §etiny dusts§r, sometimes it's available from a §efluid solution§r. It takes some practice. Oil processing and nuclear chemistry are pretty bad about this.%n%nYou can §epress @§r to see only items from that mod. For example, if you search for §c@gregtech§r, you will only see items from GregTech.%n%nYou can §epress $ §rto see only items from a specific entry in the Ore Dictionary. For example, if you search for §c$logWood§r, you will only see wooden logs.%n%nYou can §epress U §ron an item to see its uses.
 susy.quest.db.2.title=Serene Seasons
 susy.quest.db.2.desc=You are only able to grow some crops in certain seasons (but you can change that using a §agreenhouse§r, or with automated farming), and the environment around you will change throughout the year.%n%n§aSpring §rstarts on March 1 and ends on May 30.%n§6Summer §rstarts on June 1 and ends on August 30.%n§cAutumn §rstarts on September 1 and ends on November 30.%n§bWinter §rstarts on December 1 and ends on February 30.
 susy.quest.db.3.title=Picking up Rocks
-susy.quest.db.3.desc=§7Rocks§r are needed to get yourself some tools, for now.%n%nYou can get rocks by §6[Right Click]§r surface rocks that spread across the ground. In addition to being the source of rocks, §7Surface Rocks§r also serve as §6indicators of ore veins§r beneath them.
+susy.quest.db.3.desc=§7Rocks§r are needed to get yourself some tools, for now.%n%nYou can get rocks by §6[right-click]§r surface rocks that spread across the ground. In addition to being the source of rocks, §7Surface Rocks§r also serve as §6indicators of ore veins§r beneath them.
```

Comment:
Why not "right-click*ing*"?

Reply:
good catch, I did those changes with find+replace so didn't see this.

Reply:
fixed now